opensrp-chw/src/nacp/assets/json.form-sw/referrals/hiv_referral_form.json

```diff
@@ -87,16 +87,16 @@
               }
             },
             {
-              "name": "Other_symptom",
```

Comment:
@zk497 The reasons for registrations have not been updated for ctc referral form.
Please update this as per the CBHS followup visit form -> Health Challenges that had ** which required prompt for referral